src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,96 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { PageGhangeCallback } from '../../types/PageChangeCallback';
```

Comment:
i think i made a mistake in ur syntax
```suggestion
import { PageСhangeCallback } from '../../types/PageChangeCallback';
```